stempy/image.cpp

```diff
@@ -269,6 +270,62 @@ vector<STEMImage> createSTEMImages(InputIt first, InputIt last,
   return images;
 }
 
+// returns the binding result
+std::vector<double> getContainer(const STEMImage& inImage, const int numBins)
+{
+  // information about input STEMImage
+  int width = inImage.width;
+  int height = inImage.height;
+  auto curData = inImage.data;
+  std::cout << "Input STEM Image has width = " << width
```

Comment:
Might be useful while developing, but this should be removed before we merge.

Reply:
I have removed the extra printouts, thank you!